Fixed logic for the All and None buttons

wrmk opened the pull request:
The All button now adds all left items in one change event
The None button now keeps selected and disabled options

aykc reviewed (approved):
Why do we need to add the same code into `selectize.js` itself? Is presence in the plugin not enough?

wrmk commented:
Initially, I added the code into the `plugin` file, after that, I ran `npm run build` and this code was also added to other files.
As you can see in the project we import just `selectize.js` which contains all logic from plugins.
